helios/media/static_templates/question.html

```diff
@@ -1,10 +1,20 @@
-
 {#foreach $T.questions as question}
 <div id="q_view_{$T.question$index}">
+<h4><strong>{$T.question$index + 1}&ordf; Question</strong>
+{#if $T.admin_p}
+    {#if $T.question$index > 0}
+    <a href="javascript:question_move_up({$T.question$index});"><i title="Move up (reorder)." class="glyphicon glyphicon-arrow-up"></i></a>
```

Comment:
Will check out, but i was not seeing myself from the started. But Will take a good Care on this